### DISCUSSION TITLE
Is the "get_queue" action coming to the core integration?

### DISCUSSION THREAD

**Discussion by jpbaril** [music-assistant/support#3342] [Q&A]
Hi,
I was using this action when only the HACS integration existed. When the core integration released I switched to that new one only to realise it's not on par with the HACS one. So I switched back to HACS integration. But now I constantly get a message telling me to switch to the core integration because the HACS one will apparently stop working soon.

Is it planned to make the core integration on par with the HACS one and to phase out the latter?

Thanks for making all that clearer because this is a bit confusing.

**Comment by OzGav** [comment]
https://github.com/orgs/music-assistant/discussions/3259#discussioncomment-11489397
